README.md

```diff
@@ -1,2 +1,155 @@
-# Mobile-Device-Programming-Animal-Villa
-Mobile Device Programming Project
+# Animal Villa
```

Comment:
Added a Readme.md to the main branch to explain the logic of the App. This was missing. I used this to understand the App as most of the code is missing/ commented. I used the design draft and the class diagram explanation to make a sense of the different packages and tried to set up a skeleton code in next changes. Need for ReadMe: https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/